.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "aws-c-common"]
+	path = aws-c-common
+	url = git://github.com/awslabs/aws-c-common.git
```

Comment:
don't we want to checkout a tagged version?

Reply:
it's set in the submodule itself